app/src/main/java/io/github/boogiemonster1o1/eyeyoureadyforit/command/StatsCommand.java

```diff
@@ -0,0 +1,124 @@
+package io.github.boogiemonster1o1.eyeyoureadyforit.command;
+
+import discord4j.core.event.domain.interaction.SlashCommandEvent;
+import discord4j.core.object.command.ApplicationCommandInteractionOption;
+import discord4j.core.object.command.ApplicationCommandInteractionOptionValue;
+import discord4j.core.object.entity.User;
+import discord4j.core.spec.EmbedCreateSpec;
+import discord4j.discordjson.json.WebhookExecuteRequest;
+import discord4j.rest.util.Color;
+import discord4j.rest.util.WebhookMultipartRequest;
+import io.github.boogiemonster1o1.eyeyoureadyforit.App;
+import io.github.boogiemonster1o1.eyeyoureadyforit.data.GuildStatistic;
+import io.github.boogiemonster1o1.eyeyoureadyforit.data.Leaderboard;
+import io.github.boogiemonster1o1.eyeyoureadyforit.data.UserStatistic;
+import io.github.boogiemonster1o1.eyeyoureadyforit.util.StatisticsManager;
+import reactor.core.publisher.Mono;
+
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.stream.Collectors;
+
+public class StatsCommand {
+
+	public static Mono<?> handle(SlashCommandEvent event) {
+
+		if (event.getOption("server").isPresent()) return handleGuildStatsCommand(event);
```

Comment:
dont use single line ifs and pls remove extra newlines

Reply:
okie